### DISCUSSION TITLE
DatePicker

### DISCUSSION THREAD

**Discussion by phill-85** [patternfly-yew/patternfly-yew#16] [General]
I'm new to Yew, used seed a bit,  but it seems like thats been abandoned now.

I would like to add the date picker but have a few questions. 

Given (in https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/DatePicker/date-picker.hbs):

`<div class="pf-c-date-picker{{#if date-picker--modifier}} {{date-picker--modifier}}{{/if}}"
  {{#if date-picker--attribute}}
    {{{date-picker--attribute}}}
  {{/if}}>
  {{> date-picker-input}}
  {{#if date-picker-helper-text--text}}
    {{> date-picker-helper-text}}
  {{/if}}
  {{#if date-picker--IsExpanded}}
    {{#> date-picker-calendar}}
      Calendar
    {{/date-picker-calendar}}
  {{/if}}
</div>
`

I can't seem to find where "if date-picker--modifier"  or "date-picker--attribute" comes from, or how it should be implemented.

**Comment by ctron** [comment]
Not sure what the handlebars stuff is being use for. I normally take a look at their documentation, and then focus on the HTML section: https://www.patternfly.org/v4/components/date-picker/html

I also don't think that patternfly-yew supports the date picker component (yet).

If you want to give it a try, you could do an implementation, based on the `TextInput` control: https://github.com/ctron/patternfly-yew/blob/main/src/form/input.rs

**Comment by phill-85** [comment]
So is PlatternFly just the html/CCS and we're supposed to implement the logic to make it work?

**Reply by ctron** [reply]
So plain patternfly is "just" CSS and some assets (images, fonts, …). The HTML gets generated by "someone". Patternfly Yew help creating the HTML using Yew. Just like Patternfly ReactJS and ReactJS.

Or like bootstrap.

**Reply by phill-85** [reply]
Thanks. I've given up on the date picker for now. Just gone with the standard html date picker
